src/coreclr/nativeaot/Runtime/CommonMacros.h

```diff
@@ -219,6 +219,21 @@ typedef uint8_t CODE_LOCATION;
     _Pragma(FCALL_XSTRINGIFY(comment (linker, FCALL_DECL_ALTNAME(FCALL_METHOD_NAME_((__VA_ARGS__)), FCALL_ARGHELPER_STACKSIZE(__VA_ARGS__)))))
 #define FCIMPL_RENAME(_rettype, ...) \
     _Pragma(FCALL_XSTRINGIFY(comment (linker, FCALL_IMPL_ALTNAME(FCALL_METHOD_NAME_((__VA_ARGS__)), FCALL_ARGHELPER_STACKSIZE(__VA_ARGS__)))))
+#define FCIMPL_RENAME_ARGSIZE(_rettype, _method, _argSize) \
+    _Pragma(FCALL_XSTRINGIFY(comment (linker, FCALL_IMPL_ALTNAME(_method, _argSize))))
+
+#define FCIMPL1_D(_rettype, _method, a) \
+    FCIMPL_RENAME_ARGSIZE(_rettype, _method, 8) \
+    EXTERN_C _rettype REDHAWK_CALLCONV _method (a) \
```

Comment:
Could you please rename `REDHAWK_CALLCONV` to `F_CALL_CONV` while you are on it? (Aligns the name with regular CoreCLR.)

Reply:
Done